pyproject.toml

```diff
@@ -13,7 +13,8 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-mozci = "^2.0.3"
+#mozci = "^2.0.3"
+mozci = { path = "../mozci/", develop = true, extras = ['cache'] }
```

Comment:
I do this often, please make sure you reset this before you submit for final review.

Reply:
@jmaher I believe @ahal assisted me with the changes some time ago. Do you mean revert it back to 
`mozci = "^2.0.3"`?

Reply:
if these are intentional for production, then I am fine.  I often make small changes to get something running locally or to debug something and forget it when I upload the PR.  I assumed that is what this is.  If these are intentional changes, then ignore my comment